liquidctl/cli.py

```diff
@@ -352,6 +419,11 @@ def log_error(err, msg, append_err=False, *args):
     if errors:
         sys.exit(errors)
 
+    if args['--json']:
+        print(json.dumps(obj_buf, ensure_ascii=(os.getenv('LANG', None) == 'C')))
```

Comment:
Can we add a `--beautify` flag to for the json output and if it is set  then include `indent=4` as one of the parameters? For the sake of crazy people like me who sometimes want to look at the json output?

Reply:
Do you think piping to `python -m json.tool` (included in Python) or `jq` would be sufficient?  As usual, I would prefer to *not* add more options.

Reply:
oh ya that is definitely sufficient, I did not know `python -m json.tool` was a thing (it might be work mentioning that in the docs)